BioChem/UsersGuide/References.mo

```diff
@@ -0,0 +1,7 @@
+within BioChem.UsersGuide;
+
+class References
+  extends Modelica.Icons.References;
+  annotation(
+    Documentation(info = "<html><head></head><body><h2>References</h2><p><b>BioChem - A Biological and Chemical Library for Modelica (Conference paper)</b><br>Emma Larsdotter Nilsson, Peter Fritzson. Proceedings of the 3rd International Modelica Conference (November 3-4, Linköping, Sweden) 2003, pp. 215-220&nbsp;<br><br><b>A minimal cascade model for the mitotic oscillator involving cyclin and cdc2 kinase</b><br>Albert Goldbeter. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 9107-9111, October 1991 Cell Biology&nbsp;<br><br><b>Hierarchical modeling of diabetes</b><br>Elin Nyman. ISRN:LiU-IKE-EX-09/14. Linköping University 2009.&nbsp;<br><br><b>A Minimal Generic Model of Bacteria-Induced Intracellular Ca<sup>2+</sup>&nbsp;Oscillations in Epithelial Cells</b><br>Camilla Oxhamre, Agneta Richter-Dahlfors, Vladimir P. Zhdanov, and Bengt Kasemo. Biophysical Journal Volume 88 April 2005 2976-2981.&nbsp;<br><br><b>A mathematical model of metabolic insulin signaling.</b><br>A. Sedaghat, R, A. Sherman, and J. Quon, Michael. American Journal of Physiology - Endocrinology and Metabolism, 283:1048-1101, Jul 2002.&nbsp;<br><br><b>Modeling the cell division cycle: cdc2 and cyclin interactions</b><br>John J. Tyson. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 7328-7332, August 1991 Cell Biology&nbsp;<br><br><b>Modeling Feedback Loops of the Mammalian Circadian Oscillator</b><br>Becker-Weimann S, Wolf J, Herzel H, Kramer A. Biophysical Journal Volume 87 November 2004 3023-3034&nbsp;</p></body></html>"));
```

Comment:
Since it's HTML, perhaps add links to the papers while changing things?

Reply:
Done